### DISCUSSION TITLE
State of the "anounced" HAIER cooperation or are they still fighting back ? on the verge of buying a HAIER heatpump

### DISCUSSION THREAD

**Discussion by typxxi** [Andre0512/hon#248] [General]
We are on the edge of buying a Haier heatpump in particular HPM12-Nd2(GN).

After the whole mess from januar 2024 at least in Europe can the HAIER brand still be recommanded ?

Waves were high back then in January but what is the result, a real cooperation or a cat and mouse game or are the building new walls behind the scene ?

I am quite unfamiliar with the brand, never bought one and the sales contact are quite good and also fast.

Therefore my question for an update cause the videos went viral and then the story seems to have been forgotten which could be a good sign, but is it really good ?

**Comment by hellcry37** [comment]
This integration is dead. My best **guess** is dev was payed by Haier to just drop it.
